src/materials/layout/grid.html

```diff
@@ -10,7 +10,7 @@
 
   * *Column* classes should indicate the number of columns you'd like to use out of the possible 12 columns per row. E.g. `.ds-col-sm-4` for three equal columns from any sizes greater than the small breakpoint.
 
-  The Design System differs by using different breakpoints and max-width for containers. Our smallest break point (Extra Small) is `<360px` instead of Bootstrap's `<375px`.
+  The Design System uses one different breakpoint and a different max-width for containers than what Bootstrap uses. Our smallest break point (Extra Small) is less than `360px` instead of Bootstrap\'s `375px`. Our max width is `1088px` instead of Bootstrap\'s `1140px`.
```

Comment:
This was to escape the regular apostrophes because it's YML and the apostrophes to convert things to code with the greater/less than signs were confusing that bottom area. Will make all "normal" apostrophes consistent.